OneSignalSDK/onesignal/src/main/java/com/onesignal/OSNotificationWorkManager.java

```diff
@@ -10,16 +10,28 @@
 import androidx.work.Worker;
 import androidx.work.WorkerParameters;
 
+import com.google.android.gms.common.util.Strings;
```

Comment:
This won't be present if someone decides to omit Google Play Services from their app for FireOS or Huawei build of their app. I recommend using `import android.text.TextUtils` instead as this is in AndroidX. Then we can use `TextUtils.isEmpty` instead of `Strings.isEmptyOrWhitespace`.

Reply:
Good catch!